src/lib.rs

```diff
@@ -58,15 +56,6 @@ use util::sha3_256;
 #[cfg(feature = "use-insecure-test-only-mock-crypto")]
 mod mock;
 
-#[cfg(feature = "use-insecure-test-only-mock-crypto")]
```

Comment:
are you deleting this altogether with no plan of replacing it? Fine with me. Nothing better than deleted code

Reply:
I think I deleted this in rage, I'll probably add it back at some point (if need be)